Cargo.toml

```diff
@@ -10,6 +10,14 @@ readme = "README.md"
 name = "mongodb"
 version = "1.1.1"
 
+exclude = [
+    "etc/**",
+    "rustfmt.toml",
+    ".evergreen/**",
+    ".gitignore",
+    "src/test/**"
```

Comment:
Should this also exclude the top-level `tests` directory?

Reply:
Oops, I forgot we had one in the driver now that we added the connection snippets. I'll add it to the list here!